src/Mvc/Mvc.Core/src/MvcOptions.cs

```diff
@@ -136,6 +136,13 @@ public MvcOptions()
     /// </summary>
     public bool SuppressOutputFormatterBuffering { get; set; }
 
+    /// <summary>
+    /// Gets or sets the flag that determines if binding is disabled for Parseable types (contains a TryParse method)
+    /// and will treat them as <see cref="ModelMetadata.IsComplexType"/>, in case,
+    /// there isn't a <see cref="System.ComponentModel.TypeConverter"/> from <see cref="string"/> registered.
+    /// </summary>
+    public bool SuppressParseableTypesBinding { get; set; }
```

Comment:
This wasn't part of the API review

Reply:
Yeah! I will need to get back to the API review because of that
https://github.com/dotnet/aspnetcore/pull/40233#discussion_r809509448